client/src/components/StockTicker/tickerBoard.css

```diff
@@ -0,0 +1,124 @@
+.table {
```

Comment:
Why are we using CSS ? Shouldn't this be in Styles Components? Or Material UI? Just curious what is happening here.

Reply:
That's probably the easiest way to make a table like that. Once we get the main functionality of the app worked out we can go back and unify how our styling is done.